lib/git_reflow/commands/refresh.rb

```diff
@@ -0,0 +1,21 @@
+desc 'refresh will merge your current branch based on the base branch'
+long_desc <<LONGTIME
+  Performs the following:\n
+  \t$ git checkout <base_branch>\n
+  \t$ git pull <remote_location> <base_branch>\n
+  \t$ git checkout <current_branch>\n
+  \t$ git pull origin <current_branch>\n
+  \t$ git push origin <current_branch>\n
+  \t$ git merge <base_branch>\n
+LONGTIME
+arg_name '[remote_location] - remote repository name to fetch updates from (origin by default), [base_branch] - branch that you want to merge with (master by default)'
+command :refresh do |c|
+  c.action do |global_options, options, args|
+    # usage: git reflow refresh [remote_location - defaults to origin] [base_branch_name - defaults to master]
+    # defaults remote_location to origin
+    remote_location = args.length < 2 ? "origin" : args[1]
+    base_branch = args.length < 1 ? "master" : args[0]
```

Comment:
According to the logic in this commit, `git reflow refresh last-feature` will fetch the latest changes from `origin/last-feautre` (`base_branch` in context here) and merge them into the current working branch.  Is that the behavior you expect?

The name of this command no longer makes sense once you add arguments to the mix.  When you say `git reflow refresh other-feature`, it reads as though you want to update the `other-feature` branch.  As an example, `git pull other-feature` does exactly what you'd expect because the `pull` verb describes the action.

Since both arguments are optional (and likely to be used less often), I think it would make more sense to see them as flags.  So `git reflow refresh -r origin -b other-feature`.  This would let us automate the documentation for the command a little better as well.